src/main/java/ch/njol/skript/effects/EffExplosion.java

```diff
@@ -78,11 +78,13 @@ public void execute(final Event e) {
 		final Number power = force != null ? force.getSingle(e) : 0;
 		if (power == null)
 			return;
-		for (final Location l : locations.getArray(e)) {
+		for (Location location : locations.getArray(e)) {
+			if (location.getWorld() == null)
```

Comment:
It's technically *possible* we could extract a location by checking whether only one world exists or looking for the default entity in the current event or whatever, but that feels like a really patchwork solution and in all other similar syntax we just skip it/return null when the world's null.